docs/topics/Gateway.md

```diff
@@ -16,14 +16,16 @@ Important note: Not all event fields are documented, in particular, fields prefi
 
 ## Payloads
 
+
 ###### Gateway Payload Structure
 
-| Field | Type                    | Description                                                                     | Present           |
+| Field | Type                    | Description                                                                     | Nulled           |
 | ----- | ----------------------- | ------------------------------------------------------------------------------- | ----------------- |
-| op    | integer                 | [opcode](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/gateway-opcodes) for the payload | Always            |
-| d     | ?mixed (any JSON value) | event data                                                                      | Always            |
-| s     | integer                 | sequence number, used for resuming sessions and heartbeats                      | Only for Opcode 0 |
-| t     | string                  | the event name for this payload                                                 | Only for Opcode 0 |
+| op    | integer                 | [opcode](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/gateway-opcodes) for the payload | Never               |
+| d     | ?mixed (any JSON value) | event data                                                                      | Never               |
+| s     | ?integer                | sequence number, used for resuming sessions and heartbeats                      | For nonzero Opcodes |
+| t     | ?string                 | the event name for this payload                                                 | For nonzero Opcodes |
```

Comment:
I should have clarified my last comment -- I meant that the `?` in the Type column should replace the `Present` column, with a note in the descriptions of `s` and `t` saying that they are only available for Opcode 0.

To explain: you previously put "non-nulled for Opcode 0" which gave this impression that *everything but* Opcode 0 was an exception to this rule -- despite Opcode 0 being the exception rather than the norm. With that said, this new way of presenting it feels just as weird to read -- `op` and `d` are seemingly specified as exceptions to the rule of being nulled by default.

I hope I've made a bit more sense now. I'm up to discussing alternative solutions, if you want.

Reply:
@LikeLakers2 yeah you make more sense now... Did I fix this?

Reply:
I think your fix works. 👍